File: Day-25/Rajalakshmi.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
      int i,num=s.size()-1,c=0;
        if(s=="")
            return 0;
        while(s[num]==' ')
        {
            num--;
        }
      for(i=num;s[i]!=' ';i--)
      {
          c++;
          if(s[i]=='\0')
              return c-1;
      }
      return c;
    }
};

File: Day-34/Rajalakshmi.py
class Solution:
    def maximum69Number (self, num: int) -> int:
        a=[]
        dig=0
        while(num):
            a.append(int(num%10))
            dig=dig+1
            num=int(num/10)

        for i in reversed(range(dig)):
            if(a[i]==6):
                a[i]=9
                break
        ans=0
        for i in range(dig):
            ans+=a[i]*pow(10,i)
        return ans
                
            
        
            